config/initializers/plantain_config.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+module Plantain
+  def config
+    @config ||= config_yaml.with_indifferent_access
+  end
+
+  private
+
+    def config_yaml
+      # This invocation will change in ruby 2.6 to
```

Comment:
👍

Reply:
Oh hey we're on ruby 2.6 now. I should update this.